src/iadmin.cpp

```diff
@@ -1624,7 +1624,7 @@ doCommand( char *cmdToken[], rodsArguments_t* _rodsArgs = 0 ) {
         return 0;
     }
     if ( strcmp( cmdToken[0], "rmgroup" ) == 0 ) {
-        generalAdmin( 0, "rm", "user", cmdToken[1],
+        generalAdmin( 0, "rm", "group", cmdToken[1],
```

Comment:
but... only when talking to a server that is 4.3.2 or greater?

Reply:
We have not historically considered the server version (as far as I know) as iCommands versions have always been tied to a particular server version.

Are we going to start spanning server versions in iCommands version 4.3.2? If not, I think the change as-is is fine?

Reply:
oh, i see - not a PRC sniff-the-server situation...   hmm...

let's think on that...

Reply:
Is there something I need to add for this?

Reply:
What does a 4.3.1 and earlier server do when it gets this request from 4.3.2+ icommands?

Reply:
The code seems to imply it'll just throw CAT_INVALID_ARGUMENT, but I'll rebuild a 4.3.1 and check.

Reply:
Confirmed that's what happens, CAT_INVALID_ARGUMENT.

Reply:
so, 4.3.2 icommands can't do this with a 4.3.1 server.

my first reaction is that we should sniff and send compatible request.